packages/mikado_reborn/src/components/Switch/Switch.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div>
+    <span
+      class="mkr__switch"
+      :class="{ 'mkr__switch--active': value }"
+      role="checkbox"
+      :aria-checked="!!value ? true : false"
+      tabindex="0"
+      @click="toggle"
+      @keydown.space.prevent="toggle"
+    >
+      <span class="mkr__switch-indicator" />
```

Comment:
I think it should be :
```suggestion
      <span class="mkr__switch__indicator" />
```